### DISCUSSION TITLE
Osu goes black screen but I can hear sound.

### DISCUSSION THREAD

**Discussion by vavalth** [ppy/osu#14583] [Q&A]
Hello, title says it all :(

Launching the game I can see the little numbers below right freezing a little bit, then screen goes black but intro sounds goes on. 
Clicking and pressing enter does not make any sounds so I guess the game doesn't respond. 

[database.log](https://github.com/ppy/osu/files/7079621/database.log)
[performance.log](https://github.com/ppy/osu/files/7079622/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7079623/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7079624/updater.log)
Adding the logs I guess that's the best way you can help me guys, thanks a lot !

**Comment by peppy** [comment]
Please provide a video. The logs unfortunately don't give us much to go by - it looks like the game is starting and running as normal. This may be a driver or overlay issue (maybe try running the game in windowed move using Alt-Enter?).

**Reply by vavalth** [reply]
Ok jesus christ, I had to get a recording program lol 
So this time after clicking everywhere and pressing enter and space bar for a lot of times I think I go true the menu and stuff but still a black screen :( 
Once I forced the close down , Osu would not even launch again. Now it goes open, black screen and closes by itself. 
Thank you for your time ! 

https://user-images.githubusercontent.com/18542953/131572947-122e15cd-869c-47a5-8be4-d715f4819340.mp4
